docs/python/README.rst

```diff
@@ -52,6 +52,11 @@ replaces *scikit-learn* to compute the predictions.
 Changes
 -------
 
+0.4.0
+^^^^^
+
+ONNX 1.5 support, C-API custom ops, Graph optimization support
```

Comment:
Is this enough or shall I add more items like Reduced package size, Added MKL-ML binaries

Reply:
Just add a link to the github release page where we write the release notes in detail. For e.g. https://github.com/Microsoft/onnxruntime/releases/tag/v0.4.0. This link doesn't exist now, but will become active once you make the release.